README.md

```diff
@@ -17,55 +71,67 @@ In your test, you can use `stub` to set the stubbed value.
 import Moxie
 import XCTest
 
-struct MockStruct: Mock {
+class ExampleClass {
+    func foo() -> String {
+        return "this is an example"
+    }
+}
+
+class MockExampleClass: ExampleClass, Mock {
     var moxie = Moxie()
 
-    func foo() -> String {
+    override func foo() -> String {
         return value(forFunction: "foo") ?? ""
     }
 }
 
-class StructTests: XCTestCase {
+class ExampleClassTests: XCTestCase {
 
-    let mock = MockStruct()
+    let mock = MockExampleClass()
 
     func testItWorks() {
-        mock.stub(function: "foo", return: "bar")
+        mock.stub(function: "foo", return: "this is a different example")
 
-        XCTAssertEqual("bar", mock.foo())
+        XCTAssertEqual("this is a different example", mock.foo())
     }
 }
-
 ```
+
 You can also stub for a specific set of arguments:
 
 ```swift
 import Moxie
 import XCTest
 
-struct MockStruct: Mockable {
+class ExampleClass {
+    func foo(id: Int, name: String) -> Bool {
+        return false
+    }
+}
+
+class MockExampleClass: ExampleClass, Mock {
     var moxie = Moxie()
 
-    func validate(id: Int, name: String) -> Bool {
-        return value(forFunction: "validate", whenCalledWith: [id, name]) ?? false
+    override func foo(id: Int, name: String) -> Bool {
```

Comment:
Thanks! I've tried to keep it clear what is [INSERT CODE HERE] and what is actually important to the README